### PR TITLE
Milo soup is finally miso soup, fixes weird half decade long naming problem.

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

A year ago I fixed this soup to no longer taste weird, I left the name assuming it was some kind of in-joke that people from whoever first introduced it and people liked it. I have asked this question before here and on other servers and nobody really knows or gets it. I'm axing it and it's finally going to be miso.

## Why It's Good For The Game

We will no longer have every coder/player/admin wonder what the fuck a milo is, comments in code over this will be left as its legacy.

## Changelog
:cl:
fix: fixed a single letter naming issue
/:cl:

### CODE REVIEW

**Comment by Kylerace** [comment]
this dude doesnt even know about milo lmao

**Comment by tralezab** [comment]
not milopilled? hm...

**Comment by interestingusernam3** [comment]
Milo soup

**Review by coiax — APPROVED** [review]
I care not for the meme, real or not.

File: code/game/objects/items/food/soup.dm [hunk 1]
```diff
@@ -149,12 +149,12 @@
...
-/obj/item/food/soup/milo
-	name = "milosoup"
+/obj/item/food/soup/miso
+	name = "misosoup"
 	desc = "The universes best soup! Yum!!!"
-	icon_state = "milosoup"
+	icon_state = "misosoup"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/protein = 8, /datum/reagent/water = 5, /datum/reagent/consumable/nutriment/vitamin = 4)
-	tastes = list("milo" = 1) // wtf is milo //i don't know either but hey i guess japanese tourists will order it
+	tastes = list("miso" = 1) // wtf is milo //i don't know either but hey i guess japanese tourists will order it. //I carshalash the cringe fixed it
```
Comment (Mothblocks) [inline code/game/objects/items/food/soup.dm:157]: remove all these comments, comments should say what code is, not what it used to be

Comment (coiax) [inline code/game/objects/items/food/soup.dm:157]:
```suggestion
	tastes = list("miso" = 1) // There used to be comments here but mothblocks made me remove them
```

Comment (Qustinnus) [reply code/game/objects/items/food/soup.dm:157]: keep the meme alive

Comment (Fikou) [reply code/game/objects/items/food/soup.dm:157]: he will not divide us

**Comment by maxymax13** [comment]
>he doesnt know

**Review by tralezab — CHANGES_REQUESTED** [review]
If you're removing a good meme you may as well also remove the "Milo Soup Fan" joke from japanese tourists

**Comment by Qustinnus** [comment]
maints dont get the joke because they're all new

File: code/game/objects/items/food/soup.dm [hunk 2]
```diff
@@ -149,12 +149,12 @@
...
-/obj/item/food/soup/milo
-	name = "milosoup"
+/obj/item/food/soup/miso
+	name = "misosoup"
 	desc = "The universes best soup! Yum!!!"
-	icon_state = "milosoup"
+	icon_state = "misosoup"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/protein = 8, /datum/reagent/water = 5, /datum/reagent/consumable/nutriment/vitamin = 4)
-	tastes = list("milo" = 1) // wtf is milo //i don't know either but hey i guess japanese tourists will order it
+	tastes = list("miso" = 1) // There used to be comments here but mothblocks made me remove them
```
Comment (Mothblocks) [inline code/game/objects/items/food/soup.dm:157]: yeah yeah Whatever

Comment (tralezab) [reply code/game/objects/items/food/soup.dm:157]: okay now close it

Comment (carshalash) [reply code/game/objects/items/food/soup.dm:157]: Blame coiax, it made me laugh.

**Comment by optimumtact** [comment]
this is a new zealand delicacy, please do not be so culturally insensitive.

**Comment by carshalash** [comment]
For what purpose.

**Comment by interestingusernam3** [comment]
Milo soup

**Comment by optimumtact** [comment]
@carshalash what do you mean? you make an onion soup and you add milo powder

**Comment by carshalash** [comment]
Is this like, a temporary meme close or something?

**Comment by Qustinnus** [comment]
no, its milo soup.

**Comment by optimumtact** [comment]
it's milo soup dude?

**Comment by coiax** [comment]
Mmm, problem is, Mr Oranges, that the soup in game is not made with malted chocolate powder plus onions.

File: code/game/objects/items/food/soup.dm (hunk 2 above)
Comment (MrDoomBringer) [inline code/game/objects/items/food/soup.dm:157]:
```suggestion
	tastes = list("miso" = 1) // wtf is milo //i don't know either but hey i guess japanese tourists will order it
```
;)

**Comment by optimumtact** [comment]
please stop messing with my cultural heritage, I don't know how much clearer I can be

**Comment by MrDoomBringer** [comment]
wtf i love oranges

**Comment by tralezab** [comment]
oranges.

**Pull request merged**